doc/telescope-file-browser.txt

```diff
@@ -96,48 +96,54 @@ fb_picker.file_browser({opts})  *telescope-file-browser.picker.file_browser()*
+                                             (default: vim.loop.cwd())
+        {cwd_to_path}       (boolean)        whether folder browser is
+                                             launched from `path` rather than
+                                             `cwd` (default: false)
+        {grouped}           (boolean)        group initial sorting by
+                                             directories and then files; uses
+                                             plenary.scandir (default: false)
+        {files}             (boolean)        start in file (true) or folder
+                                             (false) browser (default: true)
+        {add_dirs}          (boolean)        whether the file browser shows
+                                             folders (default: true)
+        {depth}             (number)         file tree depth to display,
+                                             `false` for unlimited depth
+                                             (default: 1)
+        {select_buffer}     (boolean)        select current buffer if
+                                             possible; may imply `hidden=true`
+                                             (default: false)
+        {hidden}            (boolean)        determines whether to show hidden
+                                             files or not (default: false)
+        {respect_gitignore} (boolean)        induces slow-down w/ plenary
+                                             finder (default: false, true if
+                                             `fd` available)
+        {browse_files}      (function)       custom override for the file
+                                             browser (default:
+                                             |fb_finders.browse_files|)
+        {browse_folders}    (function)       custom override for the folder
+                                             browser (default:
+                                             |fb_finders.browse_folders|)
+        {hide_parent_dir}   (boolean)        hide `../` in the file browser
+                                             (default: false)
+        {quiet}             (boolean)        surpress any notification from
+                                             file_brower actions (default:
+                                             false)
+        {dir_icon}          (string)         change the icon for a directory
+                                             (default: )
+        {dir_icon_hl}       (string)         change the highlight group of dir
+                                             icon (default: "Default")
+        {display_stat}      (boolean|table)  ordered stat; see above notes,
+                                             (default: `{ date = true, size =
+                                             true }`)
+        {hijack_netrw}      (boolean)        use telescope file browser when
```

Comment:
Docs are autogenerated from lua files, so the changed would have to be made there. For the time being, let's maybe document this on the README somewhere, as the corresponding docs for `config.lua` are not yet generated and are also not so easy to generate appropriately. `telescope.nvim` does that in quite an involved way.